packages/utils/src/instrument.ts

```diff
@@ -82,13 +77,10 @@ function instrument(type: InstrumentHandlerType): void {
  * Use at your own risk, this might break without changelog notice, only used internally.
  * @hidden
  */
-export function addInstrumentationHandler(handler: InstrumentHandler): void {
-  if (!handler || typeof handler.type !== 'string' || typeof handler.callback !== 'function') {
-    return;
-  }
```

Comment:
We can remove this guard because we are the only ones using this `addInstrumentationHandler`